test/Combinatorics/runtests.jl

```diff
@@ -0,0 +1,3 @@
+using Oscar
+
+include("SimplicialComplexes.jl")
```

Comment:
This file is not included in the main `test/runtests.jl` so the tests do not run.